### PR TITLE
Coastguard to master

### PR DESCRIPTION
**Pull request by daniel-v-k**

### CODE REVIEW

File: cmake/llvm-gcov.sh
Comment (rreinecke) [inline cmake/llvm-gcov.sh:1]: can this be removed?

File: lib/Eigen/src/Core/Assign_MKL.h
Comment (rreinecke) [inline lib/Eigen/src/Core/Assign_MKL.h:1]: Can this and the other Eigen files be removed?

Comment (rreinecke) [reply lib/Eigen/src/Core/Assign_MKL.h:1]: @daniel-v-k

**Review by rreinecke — CHANGES_REQUESTED** [review]
Likely some files can be removed. @daniel-v-k

File: lib/Eigen/src/Core/Assign_MKL.h
Comment (daniel-v-k) [reply lib/Eigen/src/Core/Assign_MKL.h:1]: Very likely yes. Will check if everything still works as expected after this merge.

**Review by rreinecke — APPROVED** [review]

**Pull request merged**